tutorials/pages/shimmerevm-setup.md

```diff
@@ -0,0 +1,68 @@
+# Setup ShimmerEVM and run Solidity Smart Contract
+
+:::info
+In this tutorial, we will setup ShimmerEVM, connect Metamask to it, get funded and deploy a smart contract in Solidity.
+:::
+
+## Prerequisites
+
+- [Metamask](https://metamask.io/) or any other wallet of your choice.
+- [Remix](https://remix-project.org/) or any other development environment of your choice.
+
+## Set Up ShimmerEVM
+
+First, let's setup our Metamask with ShimmerEVM. For this exercise, we will use the testnet ShimmerEVM. You can find all endpoints [here](https://wiki.iota.org/shimmer/smart-contracts/guide/chains_and_nodes/testnet).
+
+Add a new network on your Metamask, as shown below:
+
+- Network Name: `ShimmerEVM Testnet`
+- New RPC URL: `https://api.sc.testnet.shimmer.network/evm/jsonrpc`
+- Chain ID: `1076`
+- Currency Symbol: `SMR`
```

Comment:
Do we only use the testnet symbol `RMS` on L1?

Reply:
We can actually use anything. The symbol doesn't matter really on client's wallet. Changing to RMS.